.github/workflows/tests_t8code_linkage_parallel_debug.yml

```diff
@@ -192,7 +192,7 @@ jobs:
     - name: make install
       run: cd build_netcdf && make install -j
     - name: make check
-      run: cd build_netcdf && make check -j V=0 CFLAGS="$CFLAGS_var -Werror" CXXFLAGS="$CXXFLAGS_var -Werror"
+      run: cd build_netcdf && make check -j8 V=0 CFLAGS="$CFLAGS_var -Werror" CXXFLAGS="$CXXFLAGS_var -Werror"
```

Comment:
Instead of hard-coding the number of jobs in each make call I suggest to set the environment variable `MAKEFLAGS` globally for all workflows.
```
  env:
    MAKEFLAGS: "-j2"
```

There are multiple hacky ways to get the number of avaible cores:
https://github.com/SimenB/github-actions-cpu-cores
https://github.com/orgs/community/discussions/25057


But it seems that only two virtual cores are available anyway. So restriciting to two jobs should be the optimum. See this for more info: https://docs.github.com/de/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
You can set `env` variables at the top level of a runner file:
https://snyk.io/de/blog/how-to-use-github-actions-environment-variables/